lang/cs/Org.Apache.REEF.Common/packages.config

```diff
@@ -24,4 +24,5 @@ under the License.
   <package id="Rx-Core" version="2.2.5" targetFramework="net45" />
   <package id="Rx-Interfaces" version="2.2.5" targetFramework="net45" />
   <package id="StyleCop.MSBuild" version="4.7.49.1" targetFramework="net45" developmentDependency="true" />
+  <package id="TransientFaultHandling.Core" version="5.1.1209.1" targetFramework="net45" />
```

Comment:
It's already used in .Client and .Wake, so adding it to one more project shouldn't be a major issue (not a new dependency but more use of the old one).